Cannot build on Windows - could not find `errhandlingapi` in `um`

I recently updated my [tool](https://github.com/mikemadden42/inv) that uses the sysinfo crate to use sysinfo-0.24.1.

On Windows, I'm getting this error when I build.

```
Running `rustc --crate-name sysinfo --edition=2018 C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\sysinfo-0.24.1\src\lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type rlib --crate-type cdylib --emit=dep-info,link -C embed-bitcode=no -C debuginfo=2 --cfg "feature=\"default\"" --cfg "feature=\"multithread\"" --cfg "feature=\"rayon\"" -C metadata=31ba6725549786e2 -C extra-filename=-31ba6725549786e2 --out-dir D:\a\inv\inv\target\debug\deps -L dependency=D:\a\inv\inv\target\debug\deps --extern cfg_if=D:\a\inv\inv\target\debug\deps\libcfg_if-a21575eac01a5ea9.rlib --extern libc=D:\a\inv\inv\target\debug\deps\liblibc-3d46275eaa99fcc4.rlib --extern ntapi=D:\a\inv\inv\target\debug\deps\libntapi-5dcd908891d829d7.rlib --extern once_cell=D:\a\inv\inv\target\debug\deps\libonce_cell-77685d5c95749480.rlib --extern rayon=D:\a\inv\inv\target\debug\deps\librayon-1b88a67d0f187c3c.rlib --extern winapi=D:\a\inv\inv\target\debug\deps\libwinapi-226a6e2ce2fbf093.rlib --cap-lints allow`
error[E0432]: unresolved import `winapi::um::errhandlingapi`
  --> C:\Users\runneradmin\.cargo\registry\src\github.com-1ecc6299db9ec823\sysinfo-0.24.1\src\windows\process.rs:36:17
   |
36 | use winapi::um::errhandlingapi::GetLastError;
   |                 ^^^^^^^^^^^^^^ could not find `errhandlingapi` in `um`
For more information about this error, try `rustc --explain E0432`.
error: could not compile `sysinfo` due to previous error
Caused by:
  process didn't exit successfully: `rustc --crate-name sysinfo --edition=2018 
  ```

Are there additional features I should enable?  I only have this in `Cargo.toml`:

```
[dependencies]
sysinfo = "0.24.1"
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,6 +23,7 @@ once_cell = "1.0"
 [target.'cfg(windows)'.dependencies]
 # FIXME: remove std feature once https://github.com/retep998/winapi-rs/pull/1003 has been merged.
 winapi = { version = "0.3.9", features = [
+  "errhandlingapi",
   "fileapi",
   "handleapi",
   "heapapi",

